include/net/ieee802154_radio.h

```diff
@@ -208,6 +213,16 @@ struct ieee802154_config {
 			uint32_t start;
 			uint32_t duration;
 		} rx_slot;
+
+		/** ``IEEE802154_CONFIG_ENH_ACK_PROBING`` */
+		struct {
+			bool lqi : 1;
```

Comment:
Thread specification requires only 1 or 2 of among (LQI, LM, RSSI)  to be selected, and order is to be kept. This structure doesn't warranty those requirements. However, OpenThread implementation neither, so maybe such improvement can be added in the future.